### PR TITLE
Jc fix/register

### PR DESCRIPTION
**Pull request by jcarrionramos**

### CODE REVIEW

File: src/components/Header/component/register.js [hunk 1]
```diff
@@ -9,29 +9,23 @@ const customContentStyle = {
...
-    phone: "",
-    password: "",
-    payment_method: "debit"
-  }
-}
-
 export default class RegisterModal extends Component {
   constructor(){
     super();
-    this.state = initialState;
```
Comment (xxvunborn) [inline src/components/Header/component/register.js:30]: why delete the binds, they are important

File: src/components/Header/component/register.js [hunk 2]
```diff
@@ -49,20 +47,52 @@ export default class RegisterModal extends Component {
...
+  arePasswordsEquals = () => {
+    if(this.state.user.password !== this.state.passVerification){
+      return false
+    }
+    return true
+  };
+
+  isDisabled = () => {
+    if (this.state.user.firstName === "" || this.state.user.lastName === "" ||
+     this.state.user.password === "") {
```
Comment (xxvunborn) [inline src/components/Header/component/register.js:76]: why !(function) return true, and not (function) return false?

**Review by xxvunborn — CHANGES_REQUESTED** [review]

File: src/components/Header/component/register.js [hunk 3]
```diff
@@ -95,16 +120,15 @@ export default class RegisterModal extends Component {
...
-        label="Crear Cuenta"
-        disabled={!this.state.disabled}
+        label="Confirmar"
         buttonStyle={{
           borderRadius: 0
         }}
         style={{
           boxShadow: 0
         }}
-        onTouchTap={this.handleClose}
```
Comment (xxvunborn) [inline src/components/Header/component/register.js:106]: handleClose require

File: src/components/Header/component/register.js (hunk 1 above)
Comment (jcarrionramos) [reply src/components/Header/component/register.js:30]: suck it 🖕

File: src/components/Header/component/register.js (hunk 2 above)
Comment (jcarrionramos) [reply src/components/Header/component/register.js:76]: tomorrow ✌️

File: src/components/Header/component/register.js (hunk 3 above)
Comment (jcarrionramos) [reply src/components/Header/component/register.js:106]: sorry 😭

**Review by xxvunborn — APPROVED** [review]

**Comment by jcarrionramos** [comment]
Merge?

**Pull request merged**